File: kyw0716/2주차/첫 시도/부족한 금액 계산하기.js
function solution(price, money, count) {
  let answer;
  let requirePrice = 0;
  for (let i = 1; i <= count; i++) {
    requirePrice += price * i;
  }
  if (money - requirePrice >= 0) answer = 0;
  else answer = Math.abs(money - requirePrice);

  return answer;
}

File: sk000801/부족한 금액 계산하기.java
package sk000801;


class Solution {
    public long solution(int price, int money, int count) {
        long answer=0;
        long total=0;
        for(long i=1; i<=count; i++) {
            total += i*price;
        }
       if(money<total) answer = total-money;
        return answer;
    }
}
